pkgs/development/libraries/accounts-qml-module/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitLab, pkgconfig, qmake, accounts-qt, qttools, signond, qtdeclarative, qtbase }:
+
+stdenv.mkDerivation rec {
```

Please read https://nixos.org/nixpkgs/manual/#sec-language-qt on Qt packages in nix
```suggestion
mkDerivation rec {
```

---

Done.